src/haddock/core/mandatory.yaml

```diff
@@ -0,0 +1,18 @@
+run_dir:
+  type: dir
+  title: Run directory
+  short: Folder to store the HADDOCK3 run
+  long: The new folder that will be created to save the HADDOCK3 run
+  explevel: easy
+molecules:
+  type: list
+  accept:
+    - ".pdb"
+  minitems: 1
+  maxitems: 20
```

Comment:
This 20 is what we now advertise. But in principle, e.g. in a scoring scenario, or virtual screening it could be more, correct?

Reply:
Or should these particular case use instead an ensemble of models?